packages/endomoat/test/index.spec.js

```diff
@@ -0,0 +1,51 @@
+const test = require('ava').default
+const {toEndoPolicy, ROOT_POLICY, WRITE_POLICY, ANY_POLICY} = require('../src')
+
+/**
+ * @typedef {import('@endo/compartment-mapper').PolicyItem & import('../src').RootPolicy} LavaMoatPackagePolicyItem
+ */
+
+test('toEndoPolicy - basic', (t) => {
+  /** @type {import('lavamoat-core/schema').LavaMoatPolicySchema} */
+  const lmPolicy = {
+    resources: {
+      a: {
+        packages: {
+          'b': true,
+        },
+        globals: {
+          'console': true,
+        },
+        builtins: {
+          'fs.readFile': true,
+        },
```

Comment:
note: there's also a key called "native" and it's rarely used so I don't yet know how we're going to support it.